torchvision/prototype/transforms/_transform.py

```diff
@@ -10,6 +10,10 @@
 
 
 class Transform(nn.Module):
+
+    # Class attribute defining transformed types. Other types are passed-through without any transformation
+    transformed_types: Tuple[Type, ...] = (torch.Tensor, _Feature, PIL.Image.Image)
```

Comment:
Does this approach works for you @datumbox ?

Reply:
It does. Can we make it private? I'm not sure we should expose it upfront. We can always revise it and make it public but right now it looks like an implementation detail.

Reply:
Yes, you are right, I'll update that